pylzy/lzy/api/v1/startup.py

```diff
@@ -118,6 +123,14 @@ def process_execution(
         raise
     logger.info(f"Execution completed in {time.time() - start} sec")
 
+    write_results = main_pid is None or int(main_pid) == pid
+    if not write_results:
```

Comment:
we should test exceptions here
all exceptions thrown by the `op` are written to a special output slot (exceptions slot)

Reply:
you mean we should write exceptions from subprocesses?
but there is only a single exception slot (same problem with multiple writes) and we are probably interested only in exceptions from "main" process